docs/detections/alerts-view-details.asciidoc

```diff
@@ -125,31 +155,79 @@ When gathering threat intelligence data for an alert, {elastic-sec} queries the
+
+Click **Threat intelligence** to expand the **Threat intelligence** sub-tab in the left panel. Within the sub-tab, matched threats are organized into two sections: <<threat-match-detected-section>> and <<enriched-with-threat-intelligence-section>>. Matched threats are ordered in reverse chronological order, with the most recent at the top. All mapped fields are displayed for each matched threat.
 
-[role="screenshot"]
-image::images/matched-indicator-sub-sec.png[Threat Match Detected subsection with matched file hash, 600]
+[discrete]
+[[threat-match-detected-section]]
+==== Threat match detected
+
+The Threat match detected section is solely reserved for threat indicator matches identified by an indicator match rule. Threat indicator matches are produced whenever event data matches a threat indicator field value in your indicator index. If indicator threat matches are not discovered, the section displays a message that none are available.
+
+[discrete]
+[[enriched-with-threat-intelligence-section]]
+==== Enriched with threat intelligence
+
+The Enriched with threat intelligence section shows indicator matches that {elastic-sec} found when querying the alert for fields with threat intelligence. You can use the date time picker to modify the query time frame, which looks at the past 30 days by default. 
+
+Click the **Inspect** button, located on the far right of the threat label, to view more information on the query. If threat matches are not discovered within the selected time frame, the section displays a message that none are available.
+
+NOTE: The event enrichment query uses the indices specified in the `securitySolution:defaultThreatIndex` advanced setting. For more information, refer to <<update-threat-intel-indices, Update default Elastic Security threat intelligence indices>>.
+
+[discrete]
+[[correlations-section]]
+=== Correlations 
+
+The Correlations sub-section shows how an alert is related to other alerts and offers ways to investigate related alerts. Use this information to quickly find patterns between alerts and then take action. 
+
+Click **Correlations** to expand the **Correlations** sub-tab in the left panel. The sub-tab displays the following information: 
 
-All other available threat intelligence data appears in the **Enriched with Threat Intelligence** subsection.
+* **Suppressed alerts**: Indicates that the alert was created with alert suppression, and shows how many duplicate alerts were suppressed. This section only appears if alert suppression is enabled for the rule.
+* **Alerts related by source event**: Shows the ten most recent alerts created by the same source event. This can help you find alerts with a shared origin and provide more context about the source event. Click the **Investigate in timeline** button to examine related alerts in Timeline.
```

Comment:
`Click the **Investigate in timeline** button to examine related alerts in Timeline.`
This confused me a bit. Looking at the sections above this one, I though we were describing the right panel. But the `Investigate in timeline` button does not appear in the right section, only in the left section above each table within the correlations sub-tab

Reply:
I can see how this can be confusing - I'll follow the structure that I've used in other sections and stick with primarily describing what's in the right panel. Changes incoming...

Reply:
Addressed in [92e7483](https://github.com/elastic/security-docs/pull/3816/commits/92e7483737ae677dde652ff744e29966975d4730).